notebooks/quantum-hardware/density-matrix.ipynb

```diff
@@ -886,8 +886,8 @@
     }
    ],
    "source": [
-    "rho_B = qi.partial_trace(rho_AB,[0])\n",
-    "rho_A = qi.partial_trace(rho_AB,[1])\n",
+    "rho_B = qi.partial_trace(rho_AB,[1])\n",
+    "rho_A = qi.partial_trace(rho_AB,[0])\n",
```

Comment:
In section 2, qubit `0` is specified to correspond to $B$, and `1` to $A$. So `rho_B` is obtained by tracing over qubit $A$, or `1`.

Reply:
If the same ordering is intended for $C$ and $D$ in section 5, those lines should probably be reversed, too.